Formula/apache-arrow-glib.rb

```diff
@@ -18,16 +18,24 @@ class ApacheArrowGlib < Formula
   end
 
   depends_on "gobject-introspection" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
   depends_on "pkg-config" => :build
   depends_on "apache-arrow"
   depends_on "glib"
 
   def install
+    args = std_meson_args + %W[
+      -Diconv=auto
+      -Dgio_module_dir=#{HOMEBREW_PREFIX}/lib/gio/modules
+      -Dbsymbolic_functions=false
+      -Ddtrace=false
+    ]
```

Comment:
These options are for GLib not Apache Arrow GLib.
We can use `std_meson_args` directly. (We don't need additional arguments.)